src/aind_mri_utils/file_io/masks.py

```diff
@@ -0,0 +1,238 @@
+        Vertex coordinates.
+    faces : numpy.ndarray
+        Face indices.
+
+    Returns
+    -------
+    trimesh.base.Trimesh
+        A trimesh object representing the surface mesh.
+    """
+    mesh = trimesh.Trimesh(vertices=vertices, faces=faces, process=False)
+    return mesh
+
+
+# Ensure normals point outward
+def ensure_normals_outward(mesh, verbose=True):
+    """
+    Ensure normals point outward.
+
+    Parameters
+    ----------
+    mesh : trimesh.base.Trimesh
+        Input mesh.
+
+    Returns
+    -------
+    trimesh.base.Trimesh
+        Mesh with outward-pointing normals.
+    """
+    if not mesh.is_watertight and verbose:
+        print(
+            "Warning: Mesh is not watertight. "
+            "Normal orientation may not be reliable."
+        )
+    else:
+        mesh.fix_normals()
+    return mesh
+
+
+# Smooth the mesh using Laplacian smoothing
+def smooth_mesh(mesh, iterations=10, lambda_param=0.5):
+    """
+    Smooth the mesh using Laplacian smoothing.
+
+    Parameters
+    ----------
+    mesh : trimesh.base.Trimesh
+        Input mesh.
+    iterations : int, optional
+        Number of iterations. The default is 10.
+    lambda_param : float, option
+        Number of iterations. The default is 10.
+
+    Returns
+    -------
+    trimesh.base.Trimesh
+        Smoothed mesh.
+    """
+    from trimesh.smoothing import filter_laplacian
+
+    smooth_mesh = mesh.copy()
+    filter_laplacian(smooth_mesh, lamb=lambda_param, iterations=iterations)
+    return smooth_mesh
+
+
+def repair_mesh(mesh, verbose=True):
+    """
+    repair_mesh(mesh)
+
+    Check if a mesh is watertight and repair it if necessary.
+
+    Parameters
+    ----------
+    mesh : trimesh.base.Trimesh
+        Input mesh.
+    verbose : bool, optional
+        If True, print messages. The default is True.
+
+    Returns
+    -------
+    trimesh.base.Trimesh
+        Repaired mesh.
+    """
+
+    # Check if the mesh is watertight
+    if mesh.is_watertight and verbose:
+        print("The mesh is already watertight.")
+    else:
+        if verbose:
+            print("The mesh is not watertight. Proceeding with repair...")
+
+        # Use PyMeshFix to repair the mesh
+        meshfix = MeshFix(mesh.vertices, mesh.faces)
+        meshfix.repair(
+            verbose=True, joincomp=True, remove_smallest_components=True
+        )
+
+        # Create a new trimesh object from the repaired mesh
+        mesh = trimesh.Trimesh(vertices=meshfix.v, faces=meshfix.f)
+
+        if verbose:
+            if mesh.is_watertight:
+                print("Mesh successfully repaired.")
+            else:
+                print(
+                    "Mesh repair failed. The mesh may still not be watertight."
+                )
+    return mesh
+
+
+def mask_to_trimesh(
+    nrrd_file_path, verbose=True, smoothing_iterations=10, smoothing_lambda=1
+):
+    """
+    mask_to_trimesh(nrrd_file_path,smoothing_iterations  = 10, smoothing_lambda
```

Comment:
numpy style docstrings do not have the signature in the docstring for python functions